flask_menu/ext.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Flask-Menu
+# Copyright (C) 2013, 2014, 2015, 2017 CERN
+# Copyright (C) 2017 Marlin Forbes
+# Copyright (C) 2023 Graz University of Technology.
+#
+# Flask-Menu is free software; you can redistribute it and/or modify
+# it under the terms of the Revised BSD License; see LICENSE file for
+# more details.
+
+"""Ext."""
+
+from flask import current_app, g
+
+from .menu import Menu
+
+
+class FlaskMenu:
```

Comment:
```suggestion
class Menu:
```

nit: per Flask convention, extensions usually don't keep the `Flask` prefix (e.g. `flask-limiter -> Limiter`, `flask-sqlalchemy -> SQLAlchemy`.

Of course this implies that the `flask_menu.menu.Menu` object is named something else, so maybe `MenuNode` or `MenuEntry`?

Reply:
ok, i change that.

I thought that the convention is the opposite so that the main entrypoint for the package has to be named as the package.